docs/loadflow/loadflow.md

```diff
@@ -91,6 +95,33 @@ $$v_{b_1} + s \cdot q_{svc} = V^{c}_{b_1}$$
 
 Where $s$ is the slope of the static var compensator.
 
+### Computing HVDC power flow
+
+#### LCC converters
+
+TODO
+
+#### VSC converters
+
+VSC converters are self-commutated converters that can be assimilated to generators in the loadflow. There can be two main modes to control active power flow through the line:
+- In **Active Power setpoint** mode, on one side of the line is the rectifier station, and on the other side of the line is the inverter station. The active power flows
+from the rectifier station to the inverter station is fixed and equal to a target value $P$ (AC side). The active power flow at each station at AC side is given by:
+  - $P_{rectifier}= P$
+  - $P_{inverter}= (1 - loss_{inverter}) * ((1 - loss_{rectifier}) * (P - P_{LineLoss}))$
+
+- In **AC emulation** mode, the active power flow between both stations is given by: $P = P_0 + k~(\theta_1 - \theta_2)$ 
+with $\theta_1$ and $\theta_2$ being the voltage angles at the bus connection for each converter station, and $P_0$ and $k$ being fixed parameters for the HVDC line. 
+If $P$ is positive, the converter station 1 is controller, else it is converter station 2. For example, if station 1 is controller, the
```

Comment:
@SylvestreSakti I have changed the sentence because I think it was weong. Maybe just check my rewording "If $P$ is positive, the converter station 1 is controller, else it is converter station 2. For example, if station 1 is controller, the
active power flow at each station is given by:..."

Reply:
Thank you Anne, indeed it was wrong, I should have named the angles theta_{controller} and theta_{noncontroller} but I think your rewording seems more clear.